winterfell/controllers/fileClaimController.js

```diff
@@ -29,7 +30,7 @@ module.exports = function (app) {
   });
 
   // Get a particular claim
-  app.get('/claims/:extClaimId', function (req, res) {
+  app.get('/claim/:extClaimId', auth.authenticatedOr404, function (req, res) {
```

Comment:
the RESTful convention I was aiming for was: use plural for resources that return an array of that model, and singular for resources that return a single instance of the model, so since this is returning just one claim I wanted to change to singular